src/components/Game/MultiPlayer/MultiplayerGamePlay/MultiplayerGamePlay.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect, useReducer, useState } from "react";
+import "../../SinglePlayer/PlayArea/playArea.css";
+import { useParams, useHistory } from "react-router-dom";
+import { multiplayerReducer } from "../../../../reducers/multiplayer-reducer";
+import { QuizData } from "../../../../data/quiz";
+import { Option, Question } from "../../../../data/quiz.type";
+import { useGame } from "../../../../contexts/game-context";
+import { PlayAreaProp } from "../../SinglePlayer/PlayArea/PlayArea";
+import { useMultiplayer } from "../../../../contexts/multiplayer-context";
+
+export type ACTION =
+  | { type: "CORRECT_ANSWER"; payload: { user: string; score: 5 } }
+  | { type: "WRONG_ANSWER"; payload: { user: string; score: 2 } }
+  | { type: "UPDATE_STATE"; payload: any }
+  | { type: "UPDATE_ISANSWERED_STATE"; payload: any };
```

Comment:
You can use constants for Actions here. Like - 
const CORRECT_ANSWER = "CORRECT_ANSWER"
{ type: typeof CORRECT_ANSWER, payload....... }